docs/msbuild/property-functions.md

```diff
@@ -338,7 +338,7 @@ Output:
 
 ## MSBuild TargetFramework and TargetPlatform functions
 
-MSBuild defines several functions for handling [TargetFramework and TargetPlatform properties](msbuild-target-framework-and-target-platform.md).
+MSBuild 16.8 and higher define several functions for handling [TargetFramework and TargetPlatform properties](msbuild-target-framework-and-target-platform.md).
```

Comment:
I thought that was 16.7; did I misremember?

Reply:
**define** or **defines**?

Reply:
> **define** or **defines**?

Depends on whether you're saying 16.8 and higher are just modifiers for MSBuild or whether there are multiple MSBuilds—one for each version. Either sounds reasonable to me, so I'd say either is correct.